doc/z-chap2.xml

```diff
@@ -87,6 +87,7 @@
     <#Include Label="JohnsonDigraph">
     <#Include Label="PetersenGraph">
     <#Include Label="GeneralisedPetersenGraph">
+    <#Include Label="StarDigraph">
```

Comment:
@LRacine I added this line so that your documentation would actually be linked into the manual - without it, the manual entry doesn't end up in the manual, and can't be accessed within GAP by typing `?StarDigraph`. You'll need to do the same thing for your tadpole graph in PR #423.

Reply:
Ah I see, thank you!!